<launch>

	<param name="acc_noise"       type="double" value="1e-2" />
    <param name="gyro_noise"      type="double" value="1e-4" />
    <param name="acc_bias_noise"  type="double" value="1e-6" />
    <param name="gyro_bias_noise" type="double" value="1e-8" />

	<param name="log_folder"      type="string" value="$(find uwb_loka)" />

	<node pkg="uwb_loka" type="uwb_loka_node" name="uwb_loka_node" output="screen">
		<rosparam command="load" file="$(find uwb_loka)/launch/uwb.yaml"/>
	</node> 
	
	<!-- <node pkg="tf" type="static_transform_publisher" name="base_link_laser_broadcaster" args="0 0 0 0 0 0 base_link laser 10" /> -->
	<!-- <node pkg="tf" type="static_transform_publisher" name="world_map_broadcaster" args="0 0 0 0 0 0 world map 10" /> -->
	<node pkg="rviz" type="rviz" name="rviz" output="screen" 
		args="-d $(find uwb_loka)/rviz/default.rviz" required="true">
	</node>

</launch>